app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard1WelcomeActivity.java

```diff
@@ -34,7 +34,6 @@ protected void onCreate(Bundle savedInstanceState) {
     public void start(View v) {
```

Comment:
Same as `goNext()` below. I think you can inline this :wink:

Reply:
here i can't inline the method's lines because this is a callback method for the button next. if you check the xml, the button onclick property calls this method